### DISCUSSION TITLE
如果将PullRefresh 和 InfiniteLoading 结合？

### DISCUSSION THREAD

**Discussion by momtboy** [jdf2e/nutui#2299] [Q&A]
### What problem does this feature solve?（这个功能解决的问题）
下拉刷新和无限加载的需求应该很常见，目前结合起来会有问题，希望官方能给出一个demo！

### What does the proposed API look like?（期望的 API）
整合的demo，列如 https://vant-ui.github.io/vant/#/zh-CN/list#xia-la-shua-xin

**Comment by TonyGJJ** [comment]
我也需要这样的功能。。

**Comment by eiinu** [comment]
一个 Demo：[NutUI Playground](https://nutui.jd.com/playground/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8bnV0LWluZmluaXRlLWxvYWRpbmcgdi1tb2RlbD1cImluZmluaXR5VmFsdWVcIiA6aGFzLW1vcmU9XCJoYXNNb3JlXCIgQGxvYWQtbW9yZT1cImxvYWRNb3JlXCI+XG4gICAgPG51dC1wdWxsLXJlZnJlc2ggdi1tb2RlbD1cInJlZnJlc2hcIiBAcmVmcmVzaD1cInJlZnJlc2hGdW5cIj5cbiAgICAgIDxkaXYgY2xhc3M9XCJpbmZpbml0ZUxpXCIgdi1mb3I9XCIoaXRlbSwgaW5kZXgpIGluIGRlZnVsdExpc3RcIiA6a2V5PVwiaW5kZXhcIj57e2l0ZW19fTwvZGl2PlxuICBcdDwvbnV0LXB1bGwtcmVmcmVzaD5cbiAgPC9udXQtaW5maW5pdGUtbG9hZGluZz5cbjwvdGVtcGxhdGU+XG5cbjxzY3JpcHQgc2V0dXA+XG5pbXBvcnQgeyByZWYgfSBmcm9tICd2dWUnO1xubGV0IGN5Y2xlID0gMDtcbmNvbnN0IHRhYnNWYWx1ZSA9IHJlZigwKTtcbmNvbnN0IGluZmluaXR5VmFsdWUgPSByZWYoZmFsc2UpO1xuY29uc3QgaGFzTW9yZSA9IHJlZih0cnVlKTtcbmNvbnN0IGxldHRlciA9IG5ldyBBcnJheSgyNikuZmlsbCgwKS5tYXAoKF8sIGluZGV4KSA9PiBpbmRleCArIDEpXG5jb25zdCBkZWZ1bHRMaXN0ID0gcmVmKGxldHRlcik7XG5jb25zdCBsb2FkTW9yZSA9IChkb25lKSA9PiB7XG4gIHNldFRpbWVvdXQoKCkgPT4ge1xuICAgIGRlZnVsdExpc3QudmFsdWUgPSBkZWZ1bHRMaXN0LnZhbHVlLmNvbmNhdChsZXR0ZXIpO1xuICAgIGN5Y2xlKys7XG4gICAgaWYgKGN5Y2xlID4gMikgaGFzTW9yZS52YWx1ZSA9IGZhbHNlO1xuICAgIGluZmluaXR5VmFsdWUudmFsdWUgPSBmYWxzZTtcbiAgfSwgMTAwMCk7XG59O1xuY29uc3QgcmVmcmVzaCA9IHJlZihmYWxzZSk7XG5jb25zdCByZWZyZXNoRnVuID0gKCkgPT4ge1xuICBzZXRUaW1lb3V0KCgpID0+IHtcbiAgICByZWZyZXNoLnZhbHVlID0gZmFsc2U7XG4gIH0sIDMwMDApO1xufTtcbjwvc2NyaXB0PlxuXG48c3R5bGU+XG4gIC5pbmZpbml0ZUxpIHtcbiAgICBmb250LXNpemU6IDE0cHg7XG4gICAgY29sb3I6ICMzMzM7XG4gICAgcGFkZGluZzogMTJweCAwO1xuICAgIGJvcmRlci1ib3R0b206IDFweCBzb2xpZCAjZWVlO1xuICB9XG48L3N0eWxlPlxuIn0=)

下次发版会更新到文档中

**Reply by TonyGJJ** [reply]
好的谢谢。
原来是要 将 PullRefresh 嵌套在 InfiniteLoading 里面 我之前整反了。。。
